.docker/docker-compose.dev.yml

```diff
@@ -5,6 +5,12 @@ services:
     build:
       target: dev
     image: libero/article-store:${IMAGE_TAG:-master}-dev
+    environment:
+      DATABASE_NAME: article-store
+      DATABASE_USER: user
+      DATABASE_PASSWORD: secret
+      DATABASE_HOST: db
+      DATABASE_PORT: 5432
```

Comment:
I tend to prefer connection strings. @giorgiosironi?

Reply:
> I tend to prefer connection strings. @giorgiosironi?

See: https://github.com/libero/article-store/pull/116#pullrequestreview-333301871

Reply:
:+1: